diff --git a/data/simple-scan.appdata.xml.in b/data/simple-scan.appdata.xml.in
index 1f10b94..c580da2 100644
--- a/data/simple-scan.appdata.xml.in
+++ b/data/simple-scan.appdata.xml.in
@@ -15,6 +15,7 @@
       This app uses the SANE framework to support most existing scanners.
     </p>
   </description>
+  <launchable type="desktop-id">org.gnome.SimpleScan.desktop</launchable>
   <screenshots>
     <screenshot type="default">
       <image type="source">https://gitlab.gnome.org/GNOME/simple-scan/raw/HEAD/data/screenshots/simple-scan-1.png</image>
@@ -93,17 +94,6 @@
     </release>
     <release date="2021-03-23" version="40.0">
       <description>
-        <p>
-          Updated translations.
-        </p>
-      </description>
-    </release>
-    <release date="2021-02-25" version="40.beta" type="development">
-      <description>
-        <p>
-          This is an unstable release in the 40 development series,
-          with the following changes:
-        </p>
         <ul>
           <li>Don't repeat vendor name in the device name</li>
           <li>GNOME 40 consistency updates</li>
